src/cascadia/WindowsTerminal/NonClientIslandWindow.cpp

```diff
@@ -844,7 +884,7 @@ bool NonClientIslandWindow::_IsTitlebarVisible() const
 {
     // TODO:GH#2238 - When we add support for titlebar-less mode, this should be

```

Comment:
Is this PR not finishing this particular TODO (so you should remove the comment?)